src/ERC721Drop.sol

```diff
@@ -1065,8 +1060,8 @@ contract ERC721Drop is
             _msgSender(),
             config.fundsRecipient,
             funds,
-            feeRecipient,
-            zoraFee
+            address(0),
```

Comment:
we can add the zora fee back here

Reply:
I don't think we should, since there is explicitly no fee on withdrawal anymore